site/v5/testing/testing.md

```diff
@@ -0,0 +1,96 @@
+- **executeMongock() method:** To perform the Mongock migration
+- **@TestPropertySource(properties = {"mongock.runner-type=NONE"}):** To prevent Mongock from injecting(and automatically executing) the Mongock runner bean. This is important to allow multiple Mongock runner's executions.
+
+Please follow these steps...
+### 1. Import the `mongock-springboot-junit5` dependency to your project
+Assuming you have imported already `mongock-springboot` to you project, you only need to add
+```xml
+        <dependency>
+            <groupId>io.mongock</groupId>
+            <artifactId>mongock-springboot-junit5</artifactId>
+            <scope>test</scope>
+        </dependency>
+```
+
+### 2. Add the additional dependencies to your project
+You probably need `Springboot starter test`, `JUnit5`, `Testcontainers`...
+
+### 3. Database initialization. 
+
+Although there are multiple ways of doing this, we present what we think provides a good balance easy-flexible
+<p class="text-center">
+    <img src="/images/integration-test-springboot-junit5-db-initialization.png" alt="ChangeUnit unit test">
+</p>
+
+### 4. Create the test class extending the `MongockSpringbootJUnit5IntegrationTestBase`
+
+This class, in addition to extending `MongockSpringbootJUnit5IntegrationTestBase`, it should also bring the database initialization and the application environment.
+<br /><br />
+This is an example.
+<p class="text-center">
+    <img src="/images/integration-test-springboot-junit5-test-class.png" alt="Integration test with JUnit5">
+</p>
+
+
+
+## Integration test with Springboot Runner WITHOUT Junit5
+
+In this case Mongock provides pretty much the same than in the JUnit5 case, with the exception of the `before` and `after` methods, what forces you to make these calls explicitly.
+
+Based on the previous scenario, the relevant modifications are
+1. Import the dependency `mongock-springboot-test` instead `mongock-springboot-junit5`
+1. Extend from `MongockSpringbootIntegrationTestBase` instead `MongockSpringbootJUnit5IntegrationTestBase`
+2. Explicitly call the methods `super.mongockBeforeEach()` and `super.mongockAfterEach()`
+
+The test class should look like this
+<p class="text-center">
+    <img src="/images/integration-test-springboot-test-class.png" alt="Integration test without JUnit5">
+</p>
+
+
+## Integration test with Standalone Runner
+In this case it requires much less help to develop integration tests, as the Standalone runner provides more control over the process.
+
+You just need to ensure that the same Mongock runner instance is not executed multiple times and the connectionDriver is not reused. The easiest way to achieve this is by re-creating the connectionDriver and re-building(and executing) the Mongock runner again in every test execution. This can be done by injecting the Mongock builder to your test class and override the connection driver.  
```

Comment:
Maybe we can put this in a more visible note (warning note?)

Reply:
No, that's an explanation, no a warning, plus it's too long to wrap it into a note.